shell/detail/provisioning.cattle.io.cluster.vue

```diff
@@ -1055,6 +1054,40 @@ export default {
             </div>
           </template>
         </SortableTable>
+        <PaginatedResourceTable
+          v-else
+          class="snapshots"
+          :data-testid="'cluster-snapshots-list'"
+          :schema="rke2SnapshotSchema"
+          :headers="rke2SnapshotHeaders"
+          :pagination-headers="rke2SnapshotPaginatedHeaders"
+          default-sort-by="age"
+          :table-actions="value.isRke1"
+          :search="false"
+          :namespaces="false"
+          :group-by="snapshotsGroupBy"
+          :api-filter="snapshotApiFilter"
+          :local-filter="snapshotLocalFilter"
+          context="provisioning.cattle.io.cluster"
+        >
+          <!-- Removing the group by namespace/list buttons in the middle header -->
+          <template #header-middle />
```

Comment:
We specify a groupBy (backup location), i think that the controls should still show (given it's an annotation that can be indexed)